pkg/labels/selector.go

```diff
@@ -77,9 +62,9 @@ func (a ByKey) Less(i, j int) bool { return a[i].key < a[j].key }
 // Requirement implements both set based match and exact match
 // Requirement is initialized via NewRequirement constructor for creating a valid Requirement.
 type Requirement struct {
-	key       string
-	operator  Operator
-	strValues util.StringSet
+	Key       string
```

Comment:
This should move to the API package, actually, if we're going to use it in the API. We'll need json tags, descriptions, merge tags, etc. once we copy it to the versioned APIs.